lib/usbmux/index.js

```diff
@@ -62,13 +63,25 @@ function swap16 (val) {
  * @returns {net.Socket} Connected socket instance
  */
 async function getDefaultSocket (opts = {}) {
-  const {
+  let {
     socketPath = DEFAULT_USBMUXD_SOCKET,
     socketPort = DEFAULT_USBMUXD_PORT,
     socketHost = DEFAULT_USBMUXD_HOST,
     timeout = 5000,
   } = opts;
 
+  let usbmuxdSocketAddress = process.env.USBMUXD_SOCKET_ADDRESS;
+  if (usbmuxdSocketAddress) {
+    let parsedAddr = url.parse(usbmuxdSocketAddress);
+    if (parsedAddr.protocol === 'unix:') {
+      socketPath = parsedAddr.pathname;
+    } else {
+      let ipPortPair = usbmuxdSocketAddress.split(':');
```

Comment:
should we validate first if `usbmuxdSocketAddress` string is formatted properly?